packages/-ember-data/tests/integration/relationships/belongs-to-test.js

```diff
@@ -889,53 +887,6 @@ module('integration/relationship/belongs_to Belongs-To Relationships', function
     assert.ok(comment instanceof Message, 'a comment is an instance of a message');
   });
 
-  test('relationshipsByName does not cache a factory', async function (assert) {
```

Comment:
this test covered a case where in pre 1.x versions of ember-data we stashed the factory onto relationship meta directly as the `type` field. It has not been the case that we do this since. The `type` field today is a string and other tests would fail were it not. This test no longer has value, so I removed it.

That said, it also failed because the `this` context of the test is registered as a destroyable by `setupTest` and thus cannot be torndown and then reused. We don't have any other tests like this but if we ever add one the solution is to not use `setupTest` at all for this test and to instead setupContext/teardownContext with objects we control.